index.d.ts

```diff
@@ -115,12 +115,12 @@ declare namespace Cmd {
     args?: any[]
   ): MapCmd<A>;
 
-  export function run<A extends Action>(
+  export function run<A extends Action, B extends Action>(
```

Comment:
so is this a breaking change? since it has 2 type params now.

Reply:
Maybe. 

If someone is explicitly passing a type param then they will get a type error saying that they didn't pass enough type params.

But if they are letting typescript infer `A` then no change will be necessary.

```typescript
// this works with the change
case FETCH_FOO_REQUEST:
       return loop(
         state,
         Cmd.run(apiFetchFoo, {
           successActionCreator: fetchFooSuccess,
           failActionCreator: fetchFooFailure,
         })
       )
```
```typescript
// compiler errors
case FETCH_FOO_REQUEST:
       return loop(
         state,
         // typescript: Expected 2 type arguments, but got 1.
         Cmd.run<Action>(apiFetchFoo, {
           successActionCreator: fetchFooSuccess,
           failActionCreator: fetchFooFailure,
         })
       )
```

Reply:
hm alright. to be safe, I'll probably make this a major bump. I might try to include some other breaking changes before releasing it. It shouldn't be too long before i do though.

Thanks!